itest/rust/src/builtin_tests/string/gstring_test.rs

```diff
@@ -69,12 +69,14 @@ fn string_chars() {
     // Empty tests regression from #228: Null pointer passed to slice::from_raw_parts().
     let string = GString::new();
     assert_eq!(string.chars(), &[]);
+    assert_eq!(string, GString::from([].as_slice()));
 
     let string = String::from("some_string");
     let string_chars: Vec<char> = string.chars().collect();
     let gstring = GString::from(string);
 
     assert_eq!(string_chars, gstring.chars().to_vec());
+    assert_eq!(gstring, GString::from(string_chars.as_slice()));
```

Comment:
In this case I'd say it's fine because `Vec<char>` is just explicitly declared above, and `Vec::as_slice()` is sufficiently clear 👍